OPEN_PROBLEMS/DECENTRALIZED_DATA_DELIVERY_MARKETS.md

```diff
@@ -0,0 +1,509 @@
+[(David) SilentDelivery: Practical Timed-delivery of Private Information using Smart Contracts](https://docs.google.com/document/d/1hXTP0VRmQsSFcnbxk3Z4sVmdW0CPajogJ6SXTe7zJ34/edit#heading=h.2q9trcjswmyw)	[10](https://docs.google.com/document/d/1hXTP0VRmQsSFcnbxk3Z4sVmdW0CPajogJ6SXTe7zJ34/edit#heading=h.2q9trcjswmyw)
+[(David) Bulletproofs+: Shorter Proofs for Privacy-Enhanced Distributed Ledger](https://docs.google.com/document/d/1hXTP0VRmQsSFcnbxk3Z4sVmdW0CPajogJ6SXTe7zJ34/edit#heading=h.bweiezm16dvh)	[10](https://docs.google.com/document/d/1hXTP0VRmQsSFcnbxk3Z4sVmdW0CPajogJ6SXTe7zJ34/edit#heading=h.bweiezm16dvh)
+
+**Known shortcomings of these approaches:**
+*   TBD
+
+#### Traditional approaches (close to Web 2.0 world)
+
+INSERT DESCRIPTION
+
+[(David) The multimedia blockchain: a distributed and tamper-proof media transaction framework](https://docs.google.com/document/d/1hXTP0VRmQsSFcnbxk3Z4sVmdW0CPajogJ6SXTe7zJ34/edit#heading=h.tsv8bvhhclyh)	[10](https://docs.google.com/document/d/1hXTP0VRmQsSFcnbxk3Z4sVmdW0CPajogJ6SXTe7zJ34/edit#heading=h.tsv8bvhhclyh)
+[(Alfonso) Reliable Client Accounting for P2P-Infrastructure Hybrids](https://docs.google.com/document/d/1hXTP0VRmQsSFcnbxk3Z4sVmdW0CPajogJ6SXTe7zJ34/edit#heading=h.mnagl72zaw3u)	[11](https://docs.google.com/document/d/1hXTP0VRmQsSFcnbxk3Z4sVmdW0CPajogJ6SXTe7zJ34/edit#heading=h.mnagl72zaw3u)
+
+**Known shortcomings of these approaches:**
+*   TOFILL
+
+### Known attacks to be mitigated
+
+Here we list the attacks to consider when designing a solution. These are:
+
+*   Malicious actor forces provider to spend bandwidth without issuing payment (also known as Grieving Attack)
+    *   Type: client fraud 
+    *   Consequence: bandwidth-spent 
+*   Malicious actor forces provider to pull the file from storage point (e.g. Filecoin, Cloud Storage, etc), paying the cost without the provider ever getting paid 
+    *   Type: client fraud: 
+    *   Consequence: currency is spent
+*   Provider claims it has sent the file, but actually never did 
+    *   Type: Provider fraud
+    *   Consequence: Client gets penalized without receiving the service
+*   Metering Inflation 
+    *   Providers report to have used more resources to serve content to clients than what they have actually done.
+*   Sybil / Throttle attacks
+    *   Description: A set of nodes collude against a content-provider to try and make him go bankrupt (loss of all its “SLA stake”). They all try to download content from that content provider at the same time, so that Providers start consuming the stake from the content providers. If clients are not paying for the service we take the risk of enabling this attack. 
+
+### New ideas being explored
+
+ResNetLab organized a Research Intensive Workshop on 3DMs, out of which, the following ideas, structured as RFCs emerged:
+
+* Link to RFCs:
+  * TOFILL
```

Comment:
Link to the following docs when they are merged:
- https://github.com/protocol/ResNetLab/pull/17
- https://github.com/protocol/ResNetLab/pull/16
- https://github.com/protocol/ResNetLab/pull/15